boot/bootdata/hivedef.inf

```diff
@@ -2082,66 +2082,94 @@ HKCU,"AppEvents\Schemes\Names\.Default","",0x00000002,"ReactOS Default"
-HKCU,"AppEvents\EventLabels\PrintComplete","",0x00000000,"Print Complete"
+HKCU,"AppEvents\EventLabels\PrintComplete","",0x00000000,"%EVENT_LABEL_16%"
 HKCU,"AppEvents\EventLabels\PrintComplete","DispFileName",0x00000000,"@mmsys.cpl,-5840"
-HKCU,"AppEvents\EventLabels\RestoreDown","",0x00000000,"Restore Down"
+HKCU,"AppEvents\EventLabels\RestoreDown","",0x00000000,"%EVENT_LABEL_17%"
 HKCU,"AppEvents\EventLabels\RestoreDown","DispFileName",0x00000000,"@mmsys.cpl,-5841"
-HKCU,"AppEvents\EventLabels\RestoreUp","",0x00000000,"Restore Up"
+HKCU,"AppEvents\EventLabels\RestoreUp","",0x00000000,"%EVENT_LABEL_18%"
 HKCU,"AppEvents\EventLabels\RestoreUp","DispFileName",0x00000000,"@mmsys.cpl,-5842"
-HKCU,"AppEvents\EventLabels\SystemAsterisk","",0x00000000,"Asterisk"
+HKCU,"AppEvents\EventLabels\SystemAsterisk","",0x00000000,"%EVENT_LABEL_19%"
 HKCU,"AppEvents\EventLabels\SystemAsterisk","DispFileName",0x00000000,"@mmsys.cpl,-5843"
-HKCU,"AppEvents\EventLabels\SystemExclamation","",0x00000000,"Exclamation"
+HKCU,"AppEvents\EventLabels\SystemExclamation","",0x00000000,"%EVENT_LABEL_20%"
 HKCU,"AppEvents\EventLabels\SystemExclamation","DispFileName",0x00000000,"@mmsys.cpl,-5845"
-HKCU,"AppEvents\EventLabels\SystemExit","",0x00000000,"Exit ReactOS"
+HKCU,"AppEvents\EventLabels\SystemExit","",0x00000000,"%EVENT_LABEL_21%"
 HKCU,"AppEvents\EventLabels\SystemExit","DispFileName",0x00000000,"@mmsys.cpl,-5846"
-HKCU,"AppEvents\EventLabels\SystemHand","",0x00000000,"Critical Stop"
+HKCU,"AppEvents\EventLabels\SystemHand","",0x00000000,"%EVENT_LABEL_22%"
 HKCU,"AppEvents\EventLabels\SystemHand","DispFileName",0x00000000,"@mmsys.cpl,-5847"
-HKCU,"AppEvents\EventLabels\SystemNotification","",0x00000000,"System Notification"
+HKCU,"AppEvents\EventLabels\SystemNotification","",0x00000000,"%EVENT_LABEL_23%"
 HKCU,"AppEvents\EventLabels\SystemNotification","DispFileName",0x00000000,"@mmsys.cpl,-5848"
-HKCU,"AppEvents\EventLabels\SystemQuestion","",0x00000000,"Question"
+HKCU,"AppEvents\EventLabels\SystemQuestion","",0x00000000,"%EVENT_LABEL_24%"
 HKCU,"AppEvents\EventLabels\SystemQuestion","DispFileName",0x00000000,"@mmsys.cpl,-5849"
-HKCU,"AppEvents\EventLabels\SystemStart","",0x00000000,"Start ReactOS"
+HKCU,"AppEvents\EventLabels\SystemStart","",0x00000000,"%EVENT_LABEL_25%"
 HKCU,"AppEvents\EventLabels\SystemStart","DispFileName",0x00000000,"@mmsys.cpl,-5850"
-HKCU,"AppEvents\EventLabels\WindowsLogoff","",0x00000000,"ReactOS Logoff"
+HKCU,"AppEvents\EventLabels\WindowsLogoff","",0x00000000,"%EVENT_LABEL_26%"
 HKCU,"AppEvents\EventLabels\WindowsLogoff","DispFileName",0x00000000,"@mmsys.cpl,-5852"
-HKCU,"AppEvents\EventLabels\WindowsLogon","",0x00000000,"ReactOS Logon"
+HKCU,"AppEvents\EventLabels\WindowsLogon","",0x00000000,"%EVENT_LABEL_27%"
 HKCU,"AppEvents\EventLabels\WindowsLogon","DispFileName",0x00000000,"@mmsys.cpl,-5853"
 
 ;-------------------------------- STRINGS -------------------------------
 
 [Strings]
+EVENT_LABEL_0 = "Default Beep"
```

Comment:
These are not good string names. The powercfg and desktop schemes below are arbitrary and user-defined, so are not good examples for the general case. These events have meaning so should be named according to that meaning (e.g. EVENT_LABEL_DEFAULT_BEEP)

Reply:
OK, i will change

Reply:
Do you want me to raise a new PR for that ?

Reply:
That would be fine. If you don't want to, then just take this as advice for future INF translations.

Reply:
OK sure !